### ISSUE TITLE
imgaug.augmenters.size.Scale did not give deprecated warning

### ISSUE THREAD

**Issue by Shreeyak** [aleju/imgaug#279]
I'm using v0.2.8 of imgaug, installed via pip inside a conda env - Ubuntu 16.04.

When I try to use the `imgaug.augmenters.size.Scale` augmenter, I am not given any deprecation warning. I see in the source code that it has been deprecated and resizer is to be used instead, and a warning saying the same, except I never see it terminal or jupyter notebook

**Comment by aleju** [comment]
`DeprecatedWarning`s are sadly by default hidden in python. You have to run your scripts with `python -W all filename.py` to see them (which of course nobody does). I guess I'll have to change the warning class, otherwise they are kind of pointless.

**Comment by Shreeyak** [comment]
Yeah, that would be nice. I was extremely surprised not to find the Scale()
func I was using in my code in the docs. Had to go digging around and found
it in the docs on source code.
Maybe we use the same kind of warnings used by tensorflow, pytorch and
similar software?

**Comment by aleju** [comment]
> I was extremely surprised not to find the Scale() func I was using in my code in the docs.

Hm, might have to add deprecated ones to the regular docs too.

> Maybe we use the same kind of warnings used by tensorflow, pytorch and similar software?

That sounds like a good idea. Will have to check how they do it.

**Comment by aleju** [comment]
Deprecation warnings are now no longer silent in master.
The implementation is similar to scikit-image. Just a library-specific DeprecationWarning.

**Comment by Shreeyak** [comment]
Awesome! Thank you so much!
